src/content/general/organizations/index.md

```diff
@@ -0,0 +1,145 @@
+---
+linkTitle: Organizations
+title: Organizations
+description: Explaining the organization concept in the Giant Swarm Management API, how to use organizations currently, and our future plans to make use of them.
+last_review_date: 2021-05-07
+weight: 40
+menu:
+  main:
+    parent: general
+user_questions:
+  - What is an organization?
+  - Do I have to use organizations?
+  - What's the benefit of using organizations?
+owner:
+  - https://github.com/orgs/giantswarm/teams/biscuit
+---
+
+# Organizations
+
+<div class="well disclaimer">
+
+<i class="fa fa-warning"></i> This article covers organizations as defined in the [Management API]({{< relref "/ui-api/management-api/_index.md" >}}). These are replacing the organizations as used with the [REST API]({{< relref "/ui-api/rest-api/index.md">}}). Please make sure you read the details about this transition carefully, especially the [roadmap](#roadmap) section, to understand the ramifications for you and your end users.
+
+<!-- TODO: link article about changes and migration -->
+
+</div>
+
+## Introduction
+
+Organizations are a way to manage resources like clusters and apps in a way that different entities are isolated from each other. You can use organizations to separate projects, business units, teams etc.
+
+Clusters are _owned_ by organizations. So in order to be able to manage a cluster (on the Management API level, not on the individual cluster level), access to the organization owning the cluster is required.
+
+The organization concept makes use of some well-known building blocks of Kubernetes in the [management cluster]({{< relref "/general/management-clusters/index.md" >}}), such as:
+
+- Namespaces
+- Role based access control (RBAC)
+- Specific custom resources (CRs) that tie things together and make organizations easy to manage
+- Operators which automate some management tasks
+
+Typical use cases for organizations are:
+
+- Separating clusters for development, experimentation, and testing from clusters used for staging and for production purposes.
+
+- Isolating teams, business units, or even legal entities.
+
+At Giant Swarm, for example, we run several shared installations where we allow different customers access to a single organization only, usually before they get their own installation. This way we can ensure that each customer, while using the same management cluster, can only access their own clusters and other resources.
```

Comment:
```suggestion
At Giant Swarm, for example, we run several shared installations where we allow different customers access to a single organization only, usually before they get their own installation. This way we can ensure that each customer, while using the same management cluster, can only access their own clusters and resources.
```

that "other" feels wrong as it might replace "only" in meaning when parsing the language, I now what is meant here, but I think it should also work without that word. or maybe replace with "related", which would also be correct as other resources in our MCs are always related to a cluster